Sources/Orbit/Components/Heading.swift

```diff
@@ -23,54 +25,77 @@ public struct Heading: View {
     }
 
     func text(sizeCategory: ContentSizeCategory) -> SwiftUI.Text {
-        SwiftUI.Text(verbatim: updatedLabel)
-            .orbitFont(
-                size: style.size,
-                weight: style.weight,
-                style: style.textStyle,
-                sizeCategory: sizeCategory
-            )
-            .foregroundColor(color?.value)
-    }
-
-    var updatedLabel: String {
-        switch style {
-            case .display, .title1, .displaySubtitle, .title2, .title3, .title4, .title5:   return label
-            case .title6:                                                                   return label.localizedUppercase
-        }
+        Text(
+            content,
+            size: .custom(style.size),
+            color: color?.textColor,
+            weight: style.weight,
+            lineSpacing: lineSpacing,
+            alignment: alignment,
+            accentColor: accentColor,
+            isSelectable: false
+        )
+        .text(sizeCategory: sizeCategory)
     }
 }
 
 // MARK: - Inits
 public extension Heading {
 
     /// Creates Orbit Heading component.
+    ///
+    /// - Parameters:
+    ///   - content: String to display. Supports html formatting tags `<strong>`, `<u>`, `<ref>`.
+    ///   - style: Heading style.
+    ///   - color: Font color. Can be set to `nil` and specified later using `.foregroundColor()` modifier.
```

Comment:
Hmm why do we have the parameter then? Do you want to switch to the "native" way of doing it everywhere at once?

Reply:
It is a tricky thing that I did not yet found a better solution for. While it seems it could be solved by !173, the main current issue is that our Text and Heading have a default color set to a specific Orbit color (inkDark), so that a caller does not need to remember specifying it every time with a `foregroundColor` modifier. 

Problem is, what I believe is a SwiftUI bug, that whenever `foregroundColor` is already applied once (inside the component), it cannot be overridden further (at a call site). So we cannot easily say "use default inkDark, but then let call site override it". That's why the current "duplicate" approach exists.

Reply:
I'd add that explanation to #173, if it's not already mentioned somewhere else.